staticjinja/staticjinja.py

```diff
@@ -316,7 +316,8 @@ def make_site(searchpath="templates",
               encoding="utf8",
               extensions=None,
               staticpaths=None,
-              filters=None):
+              filters=None,
+              env_kwargs={}):
```

Comment:
This is actually a bug. You never want to initialize parameters to mutable values. (See http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments) In this case it's actually safe, but it's bad practice.